sources/packages/web/src/components/students/ApplicationDetails.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <v-container>
+    <v-card>
+      <formio
+        formName="studentapplicationdetails"
+        :data="initialData"
+        @customEvent="customEventCallback"
+      ></formio>
+    </v-card>
+    <v-card>
+      <v-container>
+        <p class="category-header-large">Track your application</p>
+        <div class="progress">
+          <div
+            class="progress-bar bg-secondary"
+            :class="getTrackingClass()"
+            role="progressbar"
+            :style="progressBarStyle"
+            aria-valuemax="100"
+          >
+            {{ progressBarLabel }}
+          </div>
+        </div>
+        <v-row class="mt-1 mb-2">
+          <v-col>{{ ApplicationStatus.submitted }}</v-col>
+          <v-col
+            >{{ ApplicationStatus.inProgress }}
+            <i class="mr-2" :class="inProgressIconClass" aria-hidden="true"></i
```

Comment:
can you please elaborate on why are we using the _aria-hidden_ here?

Reply:
here there is no need for `aria-hidden` removing it. https://stackoverflow.com/questions/27916258/aria-hidden-true-give-no-effect-in-bootstrap

Reply:
![image](https://user-images.githubusercontent.com/77353155/135001655-7d52881b-3ffe-494b-81b1-849185a108cf.png)